tests/subscriptions/grant-keys.unit.test.ts

```
@@ -0,0 +1,60 @@
+import { randomUUID } from "node:crypto";
+import { describe, expect, test } from "vitest";
+import {
+  assertIdempotencyKey,
+  idempotencyKeySchema,
+  IDEMPOTENCY_KEY_REGEX,
```

ESLint

[error] Replace `idempotencyKeySchema,⏎··IDEMPOTENCY_KEY_REGEX` with `IDEMPOTENCY_KEY_REGEX,⏎··idempotencyKeySchema`

(prettier/prettier)